.circleci/config.yml

```diff
@@ -27,6 +27,15 @@ jobs:
           command: |
             if [ "${CIRCLE_BRANCH}" == "master" ]; then
               cd ~/cumulus-message-adapter
+
+              export VERSION=$(jq --raw-output .version package.json)
+              curl -H \
+                "Authorization: token $GITHUB_TOKEN" \
+                -d "{\"tag_name\": \"v$VERSION\", \"target_commitsh\": \"v$VERSION\", \"name\": \"v$VERSION\"}" \
+                -H "Content-Type: application/json" \
+                -X POST \
+                https://api.github.com/repos/markdboyd/cumulus-message-adapter-js/releases |grep \"url\" |grep releases |sed -e 's/.*\(https.*\)\"\,/\1/'| sed -e 's/api/uploads/'
```

Comment:
looks like this references your fork?

Reply:
Yeah, I can update that. I wanted to see if the CI build would push the release to my fork, for testing, but I guess it only does that when building the `master` branch?